packages/dds/sequence/src/test/intervalCollection.fuzz.spec.ts

```diff
@@ -356,40 +412,81 @@ describe.skip("IntervalCollection fuzz testing", () => {
-                objectStorage: new MockStorage(),
+    function runTests(seed: number, generator: Generator<Operation, FuzzTestState>, loggingInfo?: LoggingInfo): void {
+        it(`with default config, seed ${seed}`, async () => {
+            const numClients = 3;
+            const filepath = path.join(directory, `${seed}.json`);
+
+            const containerRuntimeFactory = new MockContainerRuntimeFactory();
+            const sharedStrings = Array.from({ length: numClients }, (_, index) => {
+                const dataStoreRuntime = new MockFluidDataStoreRuntime();
+                const sharedString = new SharedString(
+                    dataStoreRuntime,
+                    String.fromCharCode(index + 65),
+                    SharedStringFactory.Attributes,
+                );
+                const containerRuntime = containerRuntimeFactory.createContainerRuntime(dataStoreRuntime);
+                const services: IChannelServices = {
+                    deltaConnection: containerRuntime.createDeltaConnection(),
+                    objectStorage: new MockStorage(),
+                };
+
+                sharedString.initializeLocal();
+                sharedString.connect(services);
+                return sharedString;
+            });
+
+            const initialState: FuzzTestState = {
+                sharedStrings,
+                containerRuntimeFactory,
+                random: makeRandom(seed),
             };
 
-            sharedString.initializeLocal();
-            sharedString.connect(services);
-            return sharedString;
+            runIntervalCollectionFuzz(
+                generator,
+                initialState,
+                { saveOnFailure: true, filepath },
+                loggingInfo,
+            );
         });
+    }
 
-        const generator = take(300, makeOperationGenerator());
+    function replayTestFromFailureFile(seed: number, loggingInfo?: LoggingInfo) {
+        const filepath = path.join(directory, `${seed}.json`);
+        let operations: Operation[];
+        try {
+            operations = JSON.parse(readFileSync(filepath).toString());
+        } catch (err: any) {
+            // Mocha executes skipped suite creation blocks, but whoever's running this suite only cares if
+            // the containing block isn't skipped. Report the original error to them from inside a test.
+            if (err.message.includes("ENOENT")) {
+                it(`with default config, seed ${seed}`, () => {
+                    throw err;
+                });
+                return;
+            }
+            throw err;
+        }
 
-        const initialState: FuzzTestState = {
-            sharedStrings,
-            containerRuntimeFactory,
-            random: makeRandom(0),
-        };
+        const generator = generatorFromArray(operations);
+        runTests(seed, generator, loggingInfo);
+    }
+
+    const testCount = 10;
```

Comment:
nit: I like these at the very top of the file since it's common to pop in here just to tweak the value before checking in.